graphql_compiler/compiler/emit_sql.py

```diff
@@ -103,6 +108,83 @@ def _find_folded_outputs(ir):
     return folded_outputs
 
 
+class XMLPathBinaryExpression(BinaryExpression):
+    """Special override of BinaryExpression used to trigger `compile_xmlpath` during compile.
+
+    This type of binary expression is used to describe the Selectable selected inside the
+    XML PATH subquery. Using an XMLPathBinaryExpression forces the compiler to produce that
+    Selectable without aliasing (aka labeling) it. This prevents the string resulting from
+    the XML PATH from having extraneous XML tags based on the alias given to the column.
+    """
+
+    pass
```

Comment:
Delete this `pass`, it's unnecessary. It only has to get added when there is nothing at a deeper indentation level after a `:`. In this case, the docstring is at the deeper indentation level, so the `pass` doesn't do anything.